pkg/sql/logictest/testdata/logic_test/temp_table

```diff
@@ -1,3 +1,6 @@
+statement error unimplemented: temporary tables are only supported experimentally
```

Comment:
tip: you can check for hints in logic tests. Grep for "HINT" in the other test files for examples.